driver/CMakeLists.txt

```
@@ -2,9 +2,13 @@ include(CMakeDependentOption)
 cmake_dependent_option(LPAC_DYNAMIC_DRIVERS "Build lpac/libeuicc driver backends as a dynamic library" OFF "LPAC_DYNAMIC_LIBEUICC" OFF)
 
 option(LPAC_WITH_APDU_PCSC "Build APDU PCSC Backend (requires PCSC libraries)" ON)
-option(LPAC_WITH_APDU_AT "Build APDU AT Backend" ON)
 option(LPAC_WITH_APDU_GBINDER "Build APDU Gbinder backend for libhybris devices (requires gbinder headers)" OFF)
 option(LPAC_WITH_APDU_QMI_QRTR "Build QMI-over-QRTR backend for Qualcomm devices (requires libqrtr and libqmi headers)" OFF)
+if (Linux)
+    option(LPAC_WITH_APDU_AT "Build APDU AT Backend" ON)
+else()
+    option(LPAC_WITH_APDU_AT "Build APDU AT Backend" OFF)
+endif()
```

```suggestion
option(LPAC_WITH_APDU_AT "Build APDU AT Backend" OFF)

if (Linux)
    set(LPAC_WITH_APDU_AT ON)
endif()
```

disabled by default, but if is linux then enabled by default

see https://cmake.org/cmake/help/latest/command/option.html
see https://cmake.org/cmake/help/latest/command/set.html